Cython/Compiler/AutoDocTransforms.py

```diff
@@ -1,3 +1,4 @@
+import inspect
```

Comment:
```suggestion
import inspect

```

tiny cleanup to separate it from Cython's internal includes. Otherwise looks good

Reply:
Thanks